### PR TITLE
Add/3564 display images terminal card reader

### LINKED ISSUE #3564: Display a preview of uploaded logo and icon on terminal card reader receipt settings page
Pre-requisite: #3595

Currently, the upload logo and icon functionality works but no previews of the uploaded images are shown. Previews are good for historic reference and correcting accidentally uploaded images.

### PR DESCRIPTION
**Pull request by dpaun1985**
Fixes #3564 

#### Changes proposed in this Pull Request
- add image preview for uploaded logo and icon on Card Reader settings  page.

#### Testing instructions
- go to Card Readers settings page
- upload a logo and an icon
- images should be displayed above the upload field
- click on save changes
- refresh the page - the images should be displayed also after saving the settings

*

-------------------

- [x] Added changelog entries to both `readme.txt` and `changelog.txt` (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) : _Add link here / 'QA Testing Not Applicable'_

### CODE REVIEW

**Comment by dpaun1985** [comment]
blocked until #3674 is approved.

**Review by kalessil — APPROVED** [review]
LGTM!

**Comment by dpaun1985** [comment]
blocked also by #3730

File: client/card-readers/settings/sections/branding-details.js
Comment (mgascam) [inline client/card-readers/settings/sections/branding-details.js:87]: Would it be possible to the user a hint of what is going on here with a spinner? I mean while the Icon and logo are being retrieved. With the current behavior, the image 'pops' in when it is ready. Maybe this is a matter of preference but I think would be nice. I also don't know if we should address it in a different issue.

Also we could handle the error while retrieving the file. Currently it shows a broken image:
Screen Shot 2022-02-07 at 11 38 31

Comment (KarlisJ) [reply client/card-readers/settings/sections/branding-details.js:87]: I agree with Miguel, at least on local environment the upload time is significant enough to make some action

Comment (dpaun1985) [reply client/card-readers/settings/sections/branding-details.js:87]: > Would it be possible to give the user a hint of what is going on here with a spinner?
During the upload, the upload button has an animation which indicates that the file is being uploaded. Indeed, during the upload, we can do some other actions, like pressing the save button. 
We could create a new issue to disable the save button until the file is fully uploaded. 
What's your opinion on this ?

Comment (mgascam) [reply client/card-readers/settings/sections/branding-details.js:87]: > We could create a new issue to disable the save button until the file is fully uploaded.
What's your opinion on this ?

Yep, I think is a nice improvement, we can tackle it in a different issue.

Comment (mgascam) [reply client/card-readers/settings/sections/branding-details.js:87]: > During the upload, the upload button has an animation which indicates that the file is being uploaded
I am afraid that my original comment remains though, since I am referring to the download of the image when the page loads, for instance.

**Review by mgascam — APPROVED** [review]
LGTM! Nice job. I did some manual tests and works as expected

**Review by mordeth — APPROVED** [review]
LGTM! 👍

**Pull request merged**
